lib/services/suppressed-violations-manager.js

```diff
@@ -0,0 +1,275 @@
+"use strict";
+
+//-----------------------------------------------------------------------------
+// Requirements
+//-----------------------------------------------------------------------------
+
+const fs = require("node:fs");
+const path = require("node:path");
+const { calculateStatsPerFile } = require("../eslint/eslint-helpers");
+
+//-----------------------------------------------------------------------------
+// Exports
+//-----------------------------------------------------------------------------
+
+/**
+ * Manages the suppressed violations.
+ */
+class SuppressedViolationsManager {
```

Comment:
Can we just call this `Suppressions` for short? I'm not sure the other two words are adding much to the understanding of what this class does.

Reply:
Renamed this SuppressionsService similar to other files under `service`